package.json

```
@@ -70,6 +70,7 @@
     "@angular-eslint/eslint-plugin-template": "20.1.1",
     "@angular-eslint/template-parser": "20.1.1",
     "@angular/animations": "20.1.4",
+    "@angular/build": "^20.1.4",
```

I would prefer fixed vrsion
```suggestion
    "@angular/build": "20.1.4",
```